test/reduce.js

```diff
@@ -547,6 +570,7 @@ function producesDifferentResultWhenMinified(result_cache, code, minify_options,
     var elapsed = Date.now();
     var unminified_result = run_code(result_cache, code, toplevel, max_timeout);
     elapsed = Date.now() - elapsed;
+    if (typeof unminified_result == "string" && unminified_result.split(/\n/, 10) == 10) return false;
```

Comment:
Certainly better than my regex. But was `.length` omitted by mistake?
```
$ node -p '"ABCBACBABCBAAB".split(/A/, 4)'
[ '', 'BCB', 'CB', 'BCB' ]
$ node -p '"ABCBACBABCBAAB".split(/A/, 4) == 4'
false
$ node -p '"ABCBACBABCBAAB".split(/A/, 4).length == 4'
true
```

Reply:
Unrelated to the possible typo above... While this logic that I had suggested limiting result lines to 10 would be fine for ufuzz cases, it might be too restrictive for regular user reported `console.log` test cases and might result in "```Can't reproduce test failure with minify options provided```". Could a check be made on the number of lines that the original (first) unminified version of the code emits and use *that* limit here rather than a hardcoded number?

Reply:
Oops! 😅

Reply:
Fixed in https://github.com/mishoo/UglifyJS2/pull/3742/commits/30a88298e76c42794812a551bfbb92b7ad396d68

Reply:
(Sorry, Internet dropped out... again.)

I replied before seeing your latest message &minus; I'm currently working your suggestion, though I hit onto something that I need to debug.